src/FileHelper/file_helper.py

```diff
@@ -0,0 +1,36 @@
+import json
+import os
+
+
+class FileHelper:
+
+    def __init__(self):
+        pass
+
+
+    @staticmethod
+    def getLineFromFile(filePath, startsWith):
+        with open(filePath, 'r') as fi:
+            for line in fi:
+                stripedLine = line.replace(' ', '')
+                if stripedLine.startswith(startsWith):
+                    return stripedLine
+
+
+    @staticmethod
+    def writeJsonFile(filePath, jsonData):
+        if os.path.exists(filePath):
+            os.remove(filePath)
+
+        with open(filePath, mode='w', encoding='utf-8') as jsonFile:
+            json.dump(jsonData, jsonFile)
+
+
+    @staticmethod
+    def readJsonFile(filePath):
+        if not os.path.exists(filePath):
+            raise Exception('no file found at : {path}'.format(path=filePath))
+        
+        with open(filePath, mode='r', encoding='utf-8') as jsonFile:
+            data = json.load(jsonFile)
+            return data
```

Comment:
Should take the opportunity to create a "utils" package in src folder with FileHelper as the first class of the new package.